.claude/skills/uts-to-swift/scripts/audit_translation.py

```
@@ -0,0 +1,251 @@
+"""
+audit_translation.py — deterministic faithfulness audit of a UTS spec against its
+generated Swift test, so the per-spec review (SKILL.md Step 7) reconciles a concrete
+extracted ledger instead of eyeballing two files.
+
+Usage:
+    python3 audit_translation.py <spec-file.md> <GeneratedTests.swift>
+
+It does ZERO semantic judgement — only mechanical extraction with regex, so the same
+inputs always give the same report:
+
+  1. Test-ID coverage. Every `**Test ID**: \`<id>\`` in the spec vs every `// UTS: <id>`
```

Ruff (0.15.21)

[warning] Invalid escape sequence: `\``

Use a raw string literal

(W605)